about.html

```diff
@@ -4,9 +4,111 @@
+                            <a class="nav-link text-center" href="#">Sponsor</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link text-center" href="#">News</a>
+                        </li>
+                        <li class="nav-item hl">
+                            <a class="nav-link text-center red" href="#">CC Campaign</a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+    </section>
+
+    <section class="container bg-light p-5 m-5 row">
+        <div class="col-md-2"></div>
+        <div class="col-md-8">
+            <h3 class="text-danger fs-1 text-center">OPEN SOURCE SOFTWARE CONFERENCE 2025</h3>
+            <div class="d-flex justify-content-center"><hr width="30px" size="3" class="mt-2 text-danger"></div>
+            <p class="text-center bg-white py-3 border">The Global Summit together the community of experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two.
+                <br>This year, we hope to expand our invitation list including organizations and individuals who want to work with us on shared projects that advance the cause of the Commons, free culture and open knowledge.
+                <br>So if you're active and engaged in the worlds of opent content and knowledge - free software advocates, Wikipedians, Open Knowledge, galleries, museums, archives, governments and foundations, lawyers, and activists - we hope you'll consider joining us this year to build a stronger, more vibrant commons together. </p>
+                <p class="text-center my-5">Please contact us per Email for any further questions about OS Software Conference 2015!
+                    <br>ossconference@osvenezuela.org</p>
+        </div>
+        <div class="col-md-2"></div>
+    </section>
+
+    <section class="container-fluid bg-white py-5 row">
+        <div class="col-md-2"></div>
+        <div class="col-md-8 text-center">
+            <h3>Open Source Software Conference 2025 Logo</h3>
+            <div class="d-flex justify-content-center"><hr width="30px" size="3" class="mt-2 text-danger"></div>
+            <p class="text-center mt-3">The logo of OS Software Conference 2015 was decided through the logo competition from 8. June to 7. July.
+                After online public voting and Selection Committee's judgin process, the logo from Anthony Martin won the prize.
+                After consultation with Mr. Martin we optimized the prize-winning-logo for this year's summit as following: 
+            </p>
+            <img src="/images/logo.svg" alt="Logo" width="400px" class="mt-5 border p-4 rounded">
+        </div>
+        <hr class="my-5">
+        <h3 class="text-center">See the past OS Software Conferences</h3>
+        <div class="d-flex justify-content-center"><hr width="30px" size="3" class="mt-2 text-danger"></div>
```

Comment:
[OPTIONAL] Kindly fix these two cards, texts need to overlay the images, background is covered with orange with a little opacity like on the design.

<img width="1100" alt="Screenshot 2023-01-07 at 6 21 40 PM" src="https://user-images.githubusercontent.com/25887253/211158471-633cd2f4-c49d-4605-943f-f452e33ac435.png">